ocpmodels/models/dimenet_plus_plus.py

```diff
@@ -71,8 +72,9 @@ def __init__(
         num_radial,
         num_before_skip,
         num_after_skip,
-        act=swish,
+        act='swish',
```

Comment:
Should be `silu` instead of `swish`, as per the [activations implemented in pytorch](https://pytorch.org/docs/stable/nn.html#non-linear-activations-weighted-sum-nonlinearity).

Reply:
The `pyg` activation_resolver implements a `swish` activation as well:
https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/resolver.py#L53

It's not clear to me why they did that instead of just renaming/using the silu built-in activation, but I kept that to make it clear it was consistent with the original implementation.

Both seem like reasonable choices!

Reply:
We're on a slightly older commit of PyG unfortunately: https://github.com/pyg-team/pytorch_geometric/blob/a7e6be49e9a5e9e3ca5cedcdcb37994a32e38335/torch_geometric/nn/resolver.py#L24, which is why the tests are failing. They added `swish` in a later commit: https://github.com/pyg-team/pytorch_geometric/commit/c7062dc95183cc9f13b517dc64b5376114f9b724.

Reply:
(Independent of this, we should update to the latest PyG, but that change is more involved.)

Reply:
Got it, updated!